src/format.rs

```diff
@@ -125,6 +129,23 @@ impl fmt::Display for BinaryBytes {
     }
 }
 
+impl fmt::Display for HumanCount {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        use fmt::Write;
+
+        let num = self.0.to_string();
```

Comment:
There's almost certainly a more efficient implementation. Let me know if I should work on this or benchmark it

Reply:
Hmm, not sure there's an obviously more efficient implementation.

I do think the backwards counting loop variable is a little confusing, how about changing the loop to just use `for (i, c) in num.chars().enumerate()` and rename your current `i` to `len` (immutable), then maybe add a binding for `let pos = len - i;` or something?

Reply:
Agreed, I think the new version is a bit simpler. Another idea is to iterate in reverse, but then that requires reversing the output string